ThunderBasics/CarouselAccessibilityElement.swift

```diff
@@ -108,10 +127,12 @@ public class CarouselAccessibilityElement: UIAccessibilityElement {
     // MARK: Accessibility
 
     override public func accessibilityIncrement() {
+        guard adjustable || accessibilityTraits.contains(.adjustable) else { return }
```

Comment:
Do we need to check for `adjustable` both here and the `accessibilityTraits` get? Won't cause any issue though!

Reply:
You're correct, I don't think we should be!